docker/compile/Dockerfile.ubuntu20.04

```diff
@@ -31,12 +31,13 @@ RUN apt-get update && apt-get -y install \
       libunwind8 \
       libcppunit-dev \
       patch \
+      python3 \
```

Comment:
trivia: this is implicit from `python3-dev` [[dependencies](https://packages.ubuntu.com/focal/python3-dev)] and `python-is-python3`

Reply:
Yeah I actually had this thought with python3-dev and python3, but had an issue when I tried it without. I was planning to revisit this though with some trial and error.